docs/src/service-workers.md

```diff
@@ -0,0 +1,327 @@
+---
+id: service-workers
+title: "Service Workers Guide"
+---
+
+
+[Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) provide a browser-native method of handling requests made by a page with the native [Fetch API (`fetch`)](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) along with other network-requested assets (like scripts, css, and images).
+
+They can act as a **network proxy** between the page and the external network to perform caching logic or can provide users with an offline experience if the Service Worker adds a [FetchEvent](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent#examples) listener.
+
+Many sites that use Service Workers simply use them as a transparent optimization technique. While users might notice a faster experience, the app's implementation is unaware of their existence. Running the app with or without Service Workers enabled appears functionally equivalent.
+
+**If your app uses Service Workers**, here's the scenarios that Playwright supports:
+
+1. Testing the page exactly like a user would experience it. This works out of the box in all supported browsers.
+1. Test your page without a Service Worker. Set [`option: Browser.newContext.serviceWorkers`] to `'block'`. You can test your page as if no Service Worker was registered.
+1. Listen for and route network traffic via Playwright, whether it comes from a Service Worker or not. In Firefox and WebKit, set [`option: Browser.newContext.serviceWorkers`] to `'block'` to avoid Service Worker network traffic entirely. In Chromium, either block Service Workers or use [`method: BrowserContext.route`].
+1. (Chromium-only) Test your Service Worker implementation itself. Use [`method: BrowserContext.serviceWorkers`] to get access to the Service Worker and evaluate there.
+
+## Disabling Service Workers
+
+Set [`option: Browser.newContext.serviceWorkers`] to `'block'`.
+
```

Comment:
```suggestion
## Disabling Service Workers

Set [`option: Browser.newContext.serviceWorkers`] to `'block'`.

```

Actually, we already have a link directly to the option, so no need for this. It would just create an extra click